packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js

```diff
@@ -53,14 +64,26 @@ const defaultConfig = {
  * @hoc
  */
 const SpotlightRootDecorator = hoc(defaultConfig, (config, Wrapped) => {
-	const {noAutoFocus} = config;
+	const {getInputTypeSetter, noAutoFocus, rootId} = config;
+	const rootNode = typeof document === 'object' && document.querySelector('#' + rootId) || document;
```

Comment:
`const rootNode = typeof document === 'object' && (document.querySelector('#' + rootId) || document);`

to avoid errors during build with prerendering, I think we can use like above. Currently, I guess `document` is not type-checked.

Reply:
@Sangwook-Lee  Good catch! Thank you.